File: test/test_iapws_g704_gases_str.f90
program test_iapws_g704_gases
    use iso_fortran_env
    use iapws
    implicit none

    integer(int32) :: heavywater
    character(len=:), pointer :: gases

    print "(A)", "***** F Test gases str in water *****"
    heavywater = 0
    gases => iapws_g704_gases2(heavywater)
    print "(A, I3)", gases
    
    print "(A)", "***** F Test gases str in heavywater *****"
    heavywater = 1
    gases => iapws_g704_gases2(heavywater)
    print "(A, I3)", gases

end program

File: test/test_iapws_g704_gases_str.c
#include <stdio.h>
#include "iapws_g704.h"


int main(void){

    char *value;
    int heavywater;

    printf("***** C Test gases str in water *****\n");
    heavywater = 0;
    value = iapws_g704_capi_gases2(heavywater);
    printf("%s\n", value);
    
    printf("***** C Test gases str in heavywater *****\n");
    heavywater = 1;
    value = iapws_g704_capi_gases2(heavywater);
    printf("%s\n", value);
    
}
